DB/Pets/Dragonflight.lua

```diff
@@ -1172,6 +1172,19 @@ local dragonflightPets = {
 			{ m = CONSTANTS.UIMAPIDS.THE_FORBIDDEN_REACH, x = 29.2, y = 53.0, n = L["Entrance to Zskera Vaults"] },
 		},
 	},
+	["Brul"] = {
+		cat = CONSTANTS.ITEM_CATEGORIES.DRAGONFLIGHT,
+		type = CONSTANTS.ITEM_TYPES.PET,
+		method = CONSTANTS.DETECTION_METHODS.SPECIAL,
+		name = L["Brul"],
+		spellId = 408110,
+		itemId = 205114,
+		creatureId = 204303,
+		chance = 35,
+		requiredAura = { [CONSTANTS.AURA.ROCKS_ON_THE_ROCKS] = true },
+		sourceText = L['Looted from "Chest of Massive Gains" which spawns after killing rare - Brullo the Strong. Loot chest while "Rocks on the Rocks" buff is active. Buff is obtained from eating food "Rocks on the Rocks"'],
```

Comment:
Maybe something like this?

```lua
sourceText = format(L["Use %s before opening %s, which spawns after %s is defeated."], L["Rocks on the Rocks"], L["Chest of Massive Gains"], L["Brullo the Strong"]),
```

I reckon if the instructions are unclear, players will consult wowhead anyway.

Reply:
wow, i shouldve tested this regardless but i scrapped something similar bc it gave an error, or warning i guess but it does indeed work